index.tsx

```diff
@@ -2373,6 +2382,12 @@ export namespace MarkdownToJSX {
       state: State
     ) => React.ReactChild
 
+
+    /**
+     * Whether to enable markdown-to-jsx's built-in sanitization.
+     */
+    sanitization: boolean
```

Comment:
I like the general concept but perhaps in line with `slugify` we should just allow you to pass a no-op sanitizer function.